lib/containers/common.pm

```diff
@@ -43,7 +44,7 @@ sub install_podman_when_needed {
             assert_script_run "apt-get -y install podman", timeout => 220;
         }
         else {
-            add_suseconnect_product('sle-module-containers') if (is_sle '>=15');
+            add_suseconnect_product('sle-module-containers') if (can_build_sle_base() && is_sle '>=15');
```

Comment:
The `can_build_sle_base()` is not needed here.

Reply:
having host opensuse in OSD breaks the test. this step should be skipped as `is_sle '>=15'` will be true even for the opensuse

Reply:
I don't think you should run openSUSE tests on OSD. I don't see this as long-time scalable. We want maintainers of openSUSE as well as SLE to take over responsibility of test failures, right? So I am convinced that this should run on an instance that is available to the openSUSE community as well so that they can help.

Reply:
@okurz i am with you but i am asked to enable it on OSD following the the work we did for centos and ubuntu coverage. the reason is because we test sle images. so in theory it is a sle test.

Reply:
> I don't think you should run openSUSE tests on OSD. I don't see this as long-time scalable. We want maintainers of openSUSE as well as SLE to take over responsibility of test failures, right? So I am convinced that this should run on an instance that is available to the openSUSE community as well so that they can help.

This is not a openSUSE test, the SUT is a SLE image.

Reply:
I know but you are running a published SLE image, not the one in development, right?

Reply:
the image is `registry.suse.de/suse/sle-15-sp3/update/cr/totest/images/suse/sle15:15.3`